src/main/java/picard/illumina/parser/readers/CbclReader.java

```diff
@@ -269,7 +272,8 @@ private void advance() {
             for (int cycle = 0; cycle < outputLengths[read]; cycle++) {
                 CycleData currentCycleData = cycleData[totalCycleCount];
 
-                if (cachedTilePosition[totalCycleCount] >= cachedTile[totalCycleCount].length) {
+                if (cachedTilePosition[totalCycleCount] >= cachedTile[totalCycleCount].length
+                        || cachedTilePosition[totalCycleCount] >= cycleData[totalCycleCount].getTileInfo().getNumClustersInTile()) {
```

Comment:
If I run a NovaSeq, do I hvae to set this to true, or does it work out of the box?